packages/components/message/src/instance.ts

```diff
@@ -26,5 +26,5 @@ export const getInstance = (id: string) => {
 export const getLastOffset = (id: string): number => {
   const { prev } = getInstance(id)
   if (!prev) return 0
-  return prev.vm.exposeProxy!.bottom
+  return prev.vm?.exposeProxy?.bottom || 0
```

Comment:
```suggestion
  return prev.vm.exposeProxy?.bottom || 0
```

Reply:
Then the judgment in another file should also be adjusted.